gz_ros2_control_demos/urdf/test_cart_velocity.xacro.urdf

```diff
@@ -160,12 +160,12 @@
 
   <gazebo>
     <!-- Joint state publisher -->
-    <plugin filename="ign_ros2_control-system" name="ign_ros2_control::IgnitionROS2ControlPlugin">
-      <parameters>$(find ign_ros2_control_demos)/config/cartpole_controller_velocity.yaml</parameters>
+    <plugin filename="gz_ros2_control-system" name="gz_ros2_control::GazeboSimROS2ControlPlugin">
+      <parameters>$(find gz_ros2_control_demos)/config/cartpole_controller_velocity.yaml</parameters>
     </plugin>
     <plugin
       filename="ignition-gazebo-imu-system"
-      name="ignition::gazebo::systems::Imu">
+      name="gz::sim::systems::Imu">
```

Comment:
I see some places you're conditionally supporting one namespace or the other, but in other places you're supporting only one.

Also note that the name is being updated, but not the filename

Reply:
https://github.com/ros-controls/gz_ros2_control/pull/67/commits/1f011b322ff1d36a9eb0a539bb53b9b778aa3c82